core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -81,6 +82,9 @@ object KafkaServer {
   }
 
   val MIN_INCREMENTAL_FETCH_SESSION_EVICTION_MS: Long = 120000
+  private val EXPONENTIAL_BACKOFF_MULTIPLIER = 2
+  private val EXPONENTIAL_BACKOFF_MAX_INTERVAL_MS = 10 * 60 * 1000  // 10 minutes
```

Comment:
Let's go with 5 minutes. Max controller failover should already be below that, even in really bad cases.

Reply:
Changed to 5 mins